manifests/r/restic/restic/0.18.1/restic.restic.installer.yaml

```
@@ -1,26 +1,28 @@
-# Created with komac v2.13.0
+# Modified with Sundry.
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
 
 PackageIdentifier: restic.restic
 PackageVersion: 0.18.1

```

```suggestion
PackageVersion: 0.18.1
Commands:
  - restic
```

Add the Command field as required by the issue.